[program:socorro-crashmover]
user = socorro
command = /data/socorro/socorro/collector/crashmover_app.py --admin.conf=/etc/socorro/crashmover.ini
environment = PYTHONPATH="/data/socorro/application:/data/socorro/thirdparty"
autorestart = true
stderr_logfile = /var/log/socorro/crashmover-stderr.log
stdout_logfile = /var/log/socorro/crashmover-stdout.log
